data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "This modified Geocoris has identical cargo capacity but is modelled off one that was the result of an alien technology being merged with a hybrid Human-Hai operating system. It has additional shield emitters in a geodesic pattern over its main cargo wings and a crucial element of the ship's system core is not manufactured, but grown from inital samples taken from the first occurrence. This means that some systems are built-in and unable to be removed, and also that it is substantially more expensive than its regular version. It is; however, significantly more survivable."
+	
+ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
+	outfits
+		"Electron Turret" 3
+		"Chameleon Anti-Missile" 2
+		"AR120 Reverse Thruster" 2
+		"Boulder Reactor"
+		"Hai Ravine Batteries"
+		"Outfits Expansion"
+		"Large Heat Shunt"
+		"Quantum Keystone"
+		"Pulse Rifle" 14
+		`"Benga" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Jump Drive"
+		Hyperdrive
+		"Failed Control System"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Chameleon Anti-Missile"
+	turret "Chameleon Anti-Missile"
+	turret "Electron Turret"
+
+mission "Hai Leaks Response 5"
+	landing
+	name "Talk to Hai authorities to <planet>"
+	description "Meet Hai authorities on <destination>."
+	source "Allhome"
+	clearance
+	destination "Hai-home"
+	to offer
+		has "Hai Leaks Response 4: done"
+
+mission "Hai Leaks Response 6"
+	name "Investigate rumours of kidnapped Hai."
+	description "It's a statistical oddity, but Sayari's friend thinks someone is kidnapping Hai. Check spaceports in nearby parts of Hai and human space for evidence of a Hai trafficking operation."
+	autosave
+	landing
+	source "Hai-home"
+	clearance
+	to offer
+		has "Hai Leaks Response 4: done"
+	on offer
+		event "secret mountaintop"
+		event "role-play fleet deception"
+		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
+		"reputation: Hai" >?= 100
+		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. While they are working on a longer-term solution Sayari has quietly asked you to look into a rumour about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist and so she would like you to be sure there's no issues lurking while the diplomatic process is working.`
+		conversation
+			`When you return to Hai-home you are met by a group of relieved diplomats.`
+			`	Of all of them Alondo seems to be the least tired and takes the time to let you know what's been going on, "Captain <last>! You've missed quite the palava. Still, I doubt you lost much by missing out on trying to explain to Sayari what a 'furry' is."`
```

Comment:
```suggestion
			`	Of all of them, Alondo seems to be the least tired and takes the time to let you know what's been going on. "Captain <last>! You've missed quite the discussion. Still, I doubt you lost much. We've spent four hours trying to explain to Sayari what a 'furry' is."`
```
Never heard of "palava", and the only definition I could find was on Urban Dictionary. Probably best to stick to words we're sure are in official dictionaries. Tried to reword that last bit to avoid the double usage of `miss`.

Reply:
Turns out it's speller "palaver"

Reply:
Suggestion updated to include correct spelling.